blockchain/proxy_test.go

```diff
@@ -1203,3 +1241,231 @@ func TestProxy_IsDataTrieMigrated(t *testing.T) {
+	httpNodeStatus := `{"data":{"metrics":{"erd_current_round":187068,"erd_epoch_number":12,"erd_highest_final_nonce":21980319,"erd_nonce":21980327,"erd_nonce_at_epoch_start":172770,"erd_nonces_passed_in_current_epoch":14253,"erd_round_at_epoch_start":172814,"erd_rounds_passed_in_current_epoch":14254,"erd_rounds_per_epoch":14400}},"error":"","code":"successful"}`
+
+	t.Run("invalid block range", func(t *testing.T) {
+		invalidFilter := &sdkCore.FilterQuery{
+			FromBlock: core.OptionalUint64{21000000, true},
+			ToBlock:   core.OptionalUint64{21001000, true},
+			ShardID:   0,
+			Topics:    nil,
+			BlockHash: nil,
+		}
+
+		statusResponseBytes := []byte(httpNodeStatus)
+
+		responseMap := map[string][]byte{
+			"https://test.org/node/status": statusResponseBytes,
+		}
+
+		httpClient := createMockClientMultiResponse(responseMap)
+		args := createMockArgsProxy(httpClient)
+		ep, _ := NewProxy(args)
+
+		res, err := ep.FilterLogs(context.Background(), invalidFilter)
+		assert.Equal(t, ErrInvalidBlockRange, err)
+		assert.Nil(t, res)
+	})
+
+	t.Run("toBlock greater than last block", func(t *testing.T) {
+		invalidFilter := &sdkCore.FilterQuery{
+			FromBlock: core.OptionalUint64{21000000, false},
```

Comment:
to solve the linter issue, maybe create a small unexported function that will create a core.OptionalUint64 so the call can be read easily